code/Control/UserDefinedFormController.php

```diff
@@ -212,6 +222,42 @@ public function generateConditionalJavascript()
         }
     }
 
+    /**
+     * Returns the maximum size uploaded files can be before they're excluded from CMS configured recipient emails
+     *
+     * @return int size in megabytes
+     */
+    public function getMaximumAllowedEmailAttachmentSize()
+    {
+        return $this->parseByteSizeString($this->config()->get('maximum_email_attachment_size'));
+    }
+
+    /**
+     * Convert file sizes with a single character for unit size to true byte count.
+     * Just as with php.ini and e.g. 128M -> 1024 * 1024 * 128 bytes.
+     * @see https://www.php.net/manual/en/faq.using.php#faq.using.shorthandbytes
+     *
+     * @param string $byteSize
+     * @return int bytes
+     */
+    public function parseByteSizeString($byteSize)
```

Comment:
I don't think this should be a public function. Also, isn't there already a framework method / internal PHP method for this?

Reply:
I looked, didn't find one.
Public so it can be tested. No other reason.

Reply:
we can test private/protected functions using `ReflectionClass`

Reply:
Yeah I know, I chose the lazy option.

Reply:
Protected it is.